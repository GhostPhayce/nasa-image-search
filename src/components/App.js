import React from "react";
import Search from "../components/Search";
import "../styles/App.css";

function App() {
  return (
    <div className="app">
      <img className="app-logo" alt="NASA logo" src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg" />
      <Search />
    </div>
  );
}

export default App;
