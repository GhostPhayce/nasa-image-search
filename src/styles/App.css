.app {
  display: flex;
  flex-direction: column;
}

.app-logo {
  max-width: 200px;
  margin: 0 auto;
}
