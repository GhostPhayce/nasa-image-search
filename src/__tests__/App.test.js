import { render } from '@testing-library/react';
import App from "../components/App";

it("renders App component", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
});
