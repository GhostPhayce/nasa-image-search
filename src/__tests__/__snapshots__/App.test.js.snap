// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      alt="NASA logo"
      class="App__logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <input
      class="Search__input"
      type="text"
    />
  </div>
</DocumentFragment>
`;

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      alt="NASA logo"
      class="App__logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <input
      class="Search__input"
      type="text"
    />
  </div>
</DocumentFragment>
`;

exports[`renders App component 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      alt="NASA logo"
      class="App__logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <input
      class="Search__input"
      type="text"
    />
  </div>
</DocumentFragment>
`;
